usecases/objects/errors.go

```diff
@@ -11,7 +11,49 @@
 
 package objects
 
-import "fmt"
+import (
+	"fmt"
+)
+
+// objects status code
+const (
+	StatusForbidden           = 403
+	StatusBadRequest          = 400
+	StatusNotFound            = 404
+	StatusInternalServerError = 500
+)
```

Comment:
I know it was already discussed during live coding session, but can that be taken from http package, which is used in the codebase already, to avoid statuses duplication?

More general method could also be created `IsStatus(code int)` to check for not that common http errors, where codes from http packages could be used.

Reply:
This codes just happen to have the same names and values as http package.
They are chosen to make it easier for developers to recognise their meanings. However they are not related to http.

The real http status codes are still returned from the open-api wrappers used in adapters/handlers/rest/handlers_objects.go

For these reasons I prefer to keep it that way for the current time.